protocol-units/execution/maptos/opt-executor/src/background/transaction_pipe.rs

```diff
@@ -134,6 +137,13 @@ impl TransactionPipe {
 							&tx_result.hash,
 							&discard_status,
 						)
+					} else {
+						tracing::info!(
+							"END TX PROCESS {} Commit tx account:{} seq_number:{}",
+							tx_result.hash,
+							tx_result.sender,
+							tx_result.seq_number
+						);
```

Comment:
Less shouty and with more consistent KV use to facilitate matching:

```suggestion
						tracing::info!(
							tx_hash = %tx_result_hash,
							sender = %tx_result.sender,
							sequence_number = %tx_result.seq_number,
							"mempool rejected transaction",
						);
```